FE/src/components/interviewSettingPage/QuestionSelectionBox/QuestionTabPanelItem.tsx

```diff
@@ -91,15 +91,23 @@ const TabPanelItem: React.FC<TabPanelItemProps> = ({
             border-radius: 0 0 1rem 0;
           `}
         >
-          <Button
+          <div
+            onClick={toggleShowSelectionOption}
             css={css`
-              margin-left: auto;
+              display: flex;
+              cursor: pointer;
+              align-items: center;
```

Comment:
[p-4]
```suggestion
              align-items: center;
              column-gap: 0.25rem;
```

토글이랑 글자 사이에 간격 약간 있으면 좋을 것 같아요~
간단한 작업이니 다음 pr에서 반영하셔도 좋습니다!